src/main/app/forms/models/whatDocuments.ts

```diff
@@ -0,0 +1,41 @@
+import { MaxLength, ValidateIf, IsDefined } from 'class-validator'
+import { IsNotBlank } from 'app/forms/validation/validators/isNotBlank'
+import { Serializable } from 'models/serializable'
+
+export class ValidationErrors {
+  static readonly OTHER_DOCUMENTS_TOO_LONG: string = 'You’ve entered too many characters'
```

Comment:
do you have the concept of default validation errors? if you do then this would belong there, i.e. 
https://github.com/hmcts/cmc-citizen-frontend/blob/master/src/main/app/forms/validation/validationErrors.ts
https://github.com/hmcts/cmc-citizen-frontend/blob/master/src/main/app/forms/validation/validationConstraints.ts

might be a candidate for extracting to a module we can share